packages/parser/lib/css_parser.mly

```diff
@@ -618,3 +618,4 @@ value:
   | v = INTERPOLATION { Variable v } /* $(Lola.value) */
   | f = loc(FUNCTION) v = loc(values) RIGHT_PAREN; { Function (f, v) } /* calc() */
   | u = URL { Uri u } /* url() */
+  | WS { Delim " " }
```

Comment:
This must be unneeded

Reply:
Yes, that's wrong and I'm afraid you need it since you deleted `value_in_prelude`

Reply:
It's redundant at the moment.